docs/static/monitoring-apis.asciidoc

```diff
@@ -142,8 +248,11 @@ Example response:
 {
     "events": {
         "in": 91,
+        "bytes_in": 900000
```

Comment:
this stat `bytes_in`, `bytes_filtered` and `bytes_out` will not make alpha3, so we should remove it